package.json

```diff
@@ -36,6 +36,7 @@
 	},
 	"devDependencies": {
 		"ava": "*",
+		"tsd-check": "^0.1.0",
```

Comment:
I just released a new version (0.2.1) which supports top-level await.

Reply:
👍 awesome! Changed it to "*"